docs/docsite/rst/inventory_guide_iocage_basics.rst

```diff
@@ -0,0 +1,137 @@
+       ungrouped:
+         hosts:
+           srv_1:
+             iocage_basejail: 'no'
+             iocage_boot: 'off'
+             iocage_ip4: '-'
+             iocage_ip4_dict:
+               ip4: []
+               msg: DHCP (not running)
+             iocage_ip6: '-'
+             iocage_jid: None
+             iocage_release: 14.2-RELEASE-p3
+             iocage_state: down
+             iocage_template: ansible_client
+             iocage_type: jail
+           srv_2:
+             iocage_basejail: 'no'
+             iocage_boot: 'off'
+             iocage_ip4: '-'
+             iocage_ip4_dict:
+               ip4: []
+               msg: DHCP (not running)
+             iocage_ip6: '-'
+             iocage_jid: None
+             iocage_release: 14.2-RELEASE-p3
+             iocage_state: down
+             iocage_template: ansible_client
+             iocage_type: jail
+           srv_3:
+             iocage_basejail: 'no'
+             iocage_boot: 'off'
+             iocage_ip4: '-'
+             iocage_ip4_dict:
+               ip4: []
+               msg: DHCP (not running)
+             iocage_ip6: '-'
+             iocage_jid: None
+             iocage_release: 14.2-RELEASE-p3
+             iocage_state: down
+             iocage_template: ansible_client
+             iocage_type: jail
+
+Notes:
+
+* Optionally, create shared IP jails
+
+.. code-block:: console
+
+   shell> iocage create --template ansible_client --name srv_1 ip4_addr="em0|10.1.0.101/24"
+   srv_1 successfully created!
+   shell> iocage create --template ansible_client --name srv_2 ip4_addr="em0|10.1.0.102/24"
+   srv_2 successfully created!
+   shell> iocage create --template ansible_client --name srv_3 ip4_addr="em0|10.1.0.103/24"
+   srv_3 successfully created!
+
+.. code-block:: console
+
+   shell> iocage list -l
+   +------+-------+------+-------+------+-----------------+-------------------+-----+----------------+----------+
+   | JID  | NAME  | BOOT | STATE | TYPE |     RELEASE     |        IP4        | IP6 |    TEMPLATE    | BASEJAIL |
+   +======+=======+======+=======+======+=================+===================+=====+================+==========+
+   | None | srv_1 | off  | down  | jail | 14.2-RELEASE-p3 | em0|10.1.0.101/24 | -   | ansible_client | no       |
+   +------+-------+------+-------+------+-----------------+-------------------+-----+----------------+----------+
+   | None | srv_2 | off  | down  | jail | 14.2-RELEASE-p3 | em0|10.1.0.102/24 | -   | ansible_client | no       |
+   +------+-------+------+-------+------+-----------------+-------------------+-----+----------------+----------+
+   | None | srv_3 | off  | down  | jail | 14.2-RELEASE-p3 | em0|10.1.0.103/24 | -   | ansible_client | no       |
+   +------+-------+------+-------+------+-----------------+-------------------+-----+----------------+----------+
+
+See: `Configuring a Shared IP Jail <https://iocage.readthedocs.io/en/latest/networking.html#configuring-a-shared-ip-jail>`_
```

Comment:
If this is part of the above note, it needs to be indented as well.

Reply:
Done.